app/api/tests/factories.py

```diff
@@ -17,6 +17,8 @@
 BASE64_OF_VALID_CA_CERT = "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUZmekNDQTJlZ0F3SUJBZ0lWQUxvT0pBb1NQMW04MUJRM0RBalJIY1lYckxSOE1BMEdDU3FHU0liM0RRRUJDd1VBTUhjeEN6QUpCZ05WQkFZVEFsVlRNUXN3Q1FZRFZRUUlFd0pEVHpFUU1BNEdBMVVFQnhNSFFtOTFiR1JsY2pFU01CQUdBMVVFQ2hNSlNuVnRjRU5zYjNWa01Sa3dGd1lEVlFRTEV4QktkVzF3UTJ4dmRXUlRRVTFNU1dSUU1Sb3dHQVlEVlFRREV4RktkVzF3UTJ4dmRXUlRRVTFNVlhObGNqQWVGdzB5TVRFeE1qa3dNVEF5TVRSYUZ3MHlOakV4TWprd01UQXlNVFJhTUhjeEN6QUpCZ05WQkFZVEFsVlRNUXN3Q1FZRFZRUUlFd0pEVHpFUU1BNEdBMVVFQnhNSFFtOTFiR1JsY2pFU01CQUdBMVVFQ2hNSlNuVnRjRU5zYjNWa01Sa3dGd1lEVlFRTEV4QktkVzF3UTJ4dmRXUlRRVTFNU1dSUU1Sb3dHQVlEVlFRREV4RktkVzF3UTJ4dmRXUlRRVTFNVlhObGNqQ0NBaUl3RFFZSktvWklodmNOQVFFQkJRQURnZ0lQQURDQ0Fnb0NnZ0lCQUxjNktKT0czTm0wMnZIZnZvYVdrcjBzUjk0SE9Wd2lLNzlqZHhQNHNhQ2k1aEw3RmoyRW5FbXo3M0JIL0J4QkZRL3VIY1JqTU85dUxuNldSY1QyUDhXRE10eVV1QlNJVUw0bEx4b1RPbTAvMzdxcllZQUhmYllKdVBXQWJ2SXhuZTJOczBpWFlGa2dIU1o2RHVkWjM3U1NkWG5QQnVSNmNhZXltYm92ckNIUGJFVGIzU3BnY1ZNdXV1RzFYaENUTjBsWi94cnBCMUc4SHFMMzd4VkNtSkF6bUJtVWdZcHU5K3pIMXVCUHdVb1dhOFRIZWxYcnAyQ1VaM210d28wdUtuZnlYSmNKeUM1ckp2MFJMbzRvSlJldFUzbWlURjcvdHJjWE1oWEdzb3NNL1UvYTVzbjc5RWgzdngrQkpDRGRySnRlNXowV0NDUitGY0xZdEU5aXdlV3BJS2g5ODc0NnJVb1M0ck1IcFVhZTBOczZlU3BVK093SW1NdzZvVUNITzgreDFna2NWQkcydGZEMG12N1RJZFc1aWI2TTlMOVQ2M0wxNXFla2U5QVBQY3BHMHZHNUl4ZUdiQ2xSY2pFNHVzaVRnK2lLOCtBQ1Q3aDJodFNjU0dsUHNJM0RibG45RDRMWFJLTkhDY3lCY3BWT0hJMDZaMEQwaEs3eWNscGl1SUxTSGFUVENQbDM4eHdVTkZsSkRxWGpVdnpMeE0xc1d6ZWJ0NEl0M2c4ODZNa1M0bDB3WmdhWUh4bWNtQ2RsSnZ5UHFWOHR4Z1FaWUJZM2pUN0VqZ1BGb3g0a0xNVktBK2pBemY5c0hUaDd6UW5PZ1JFMzJyaGoyTlVBSzNoQmJIdjFhT2VVbGh4U0xEbGU3WDZsWEd4eEhDdkEzbDFOcG1vNUExT1poTUJGQWdNQkFBR2pBakFBTUEwR0NTcUdTSWIzRFFFQkN3VUFBNElDQVFDVUlvcDJUU1FKenNSaGd3T0dZa2JwQWJsU2prTlE1VEJaZnJyWm9GWU9NQTBqaTYycWxXRDNDNU9VYVdRYkJydkcvOEx2Q09YbTRtUG1wMWUwSmVsaTZEWkJJbjJVbzduZTI5VitpdHZnQi9kdTYrcGtySXIwZWdBYmtKZmtTK2YzbFFqZXBqRmFraVFxSzNZTEp0WEpVckt2d2pXa2RnVG1XcjhTMVA5TFg0ZkU0UmxyOWkrcGc2TlZzcFNEZXptREhnN2piZ2NxMXRLOGczcmFEcEFNNExreUdKSENTRTB0V21ORHc2UUtSYi9ldjZmQmR6MVVWVFhhV1pvQTIycldjZk1IMzVZd2NDUDVvWHBpa0lTaStKbUc2SG9qQnM0bGpwYlpGWWNSUnU5UC9pMG12cGRKUXRQUlJudk5DNXY1RXdQdWt0RTFXaTZxa3A2OE43aitRTGw4anlhWExuNkdIRTZDamdnRTRZQjh2ZXFjZUxhRERZdXR4UmpUNzdMaEVTeFdONlhSQnpoTWNPckhGcE5KUUkxVmxhbEFCVzJZanBKSVB2bytpV2xBWlp4MjBrMitHRkpWTml3ZTBYemR5cWwxZUdNeENrS3BkNXdCZXpKZUJVdXJNUSt0cWQrMWRHMTBmRUJMM2dpa0JHbFpMU1d1czNwRnhTaXdZemhTU29BcUs5ekY3VCtBNjc0cDdFUUI5Zms4VjRadFI2T28yMFI0TldPWDRWcnFzekZjWWFOSnJwS3VCOEZhRHZVcWNFMmFRK3ZrWVhmaTBGYWRMRm1jN1dlTWVQSU12a2ZpbnI5cUVnWWMreXE1WGEwV0hiM1hlKzF5N2wwVEt5dUtIZEJnSEdKVUJBYkFFeWF1NFVjSUJqbjFnWDJZTlEvTjFUUnZxSWJrY1E9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0t"
```

Comment:
do we need this const in code?   can this just be `VALID_CA_CERT_1 = base64.b64_encode(...)` ?

Reply:
Yes we could technically use `BASE64_OF_VALID_CA_CERT = base64.b64_encode(...)`. IMO, I think it's nicer to assert the exact base64 string, because we want test to fail if somehow `b64_encode()` changes its implementation, e.g. add trailing padding `\n`, we want to be able to catch these changes. If we rely on `b64encode()` during the test, then the test only verifies that we call `b64encode()`.